doc/fixing-broken-sites.md

```diff
@@ -0,0 +1,51 @@
+# How to fix broken site issues
+
+Unfortunately, while working to protect your privacy, Privacy Badger can end up breaking website functionality. This document is about the process of classifying and resolving these breakages.
+
+Here are the [open "broken site" and "good-volunteer-task"-labeled issues](https://github.com/EFForg/privacybadger/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20label%3A%22broken%20site%22%20label%3A%22good%20volunteer%20task%22).
+
+
+## Confirm Privacy Badger is responsible
+
+The first thing to do is to confirm that Privacy Badger blocks (or will eventually learn to block) the domain, and that blocking the domain does indeed break the site.
+
+Does disabling Privacy Badger for the site and reloading the page fix the issue? If it doesn't, does disabling the entire Privacy Badger add-on and reloading the page fix the issue? If it still doesn't, Privacy Badger is not at fault.
+
+If disabling Badger and reloading the page fixed the issue, and re-enabling and reloading brought the issue back, let's try to figure out the responsible domain(s). Try allowing half the blocked domains to load. If (after reloading the page) the issue was fixed, pick half of those domains and revert them back to Badger's control. Eventually you should find the exact domain(s) that, when blocked, cause the issue to appear.
```

Comment:
I would note that when you re-enable PB and refresh the page, sometimes the issue might not be reproducible because the page is cached. You should note that the user should do `CTRL + F5` in order to reproduce the issue.